app/feedme/search/hybrid_search_engine.py

```
@@ -0,0 +1,414 @@
+            
+            # Get individual scores
+            vector_score = vector_scores.get(result_id, 0.0)
+            text_score = text_scores.get(result_id, 0.0)
+            
+            # Normalize text scores (ts_rank can vary widely)
+            if text_score > 0:
+                text_score = min(text_score, 1.0)  # Cap at 1.0
+            
+            # Calculate combined score
+            combined_score = (vector_score * self.vector_weight + 
+                            text_score * self.text_weight)
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)